pymks/mks_structure_analysis.py

```diff
@@ -87,7 +87,7 @@ def __init__(self, basis, correlations=None, dimension_reducer=None,
         if basis is not None:
             self.basis._n_jobs = n_jobs
         if self.dimension_reducer is None:
-            self.dimension_reducer = RandomizedPCA(copy=False)
+            self.dimension_reducer = RandomizedPCA(copy=False,svd_solver="randomized")
```

Comment:
Should this be changed to `PCA` since you're no longer importing `RandomizedPCA`?